src/birl/duration.gleam

```diff
@@ -258,8 +258,14 @@ const unit_values = [
 ]
 
 fn inner_blur(values: List(#(Int, Unit))) -> #(Int, Unit) {
-  let assert Ok(second) = list.at(values, 0)
-  let leading = list.at(values, 1)
+  let assert Ok(second) = case values {
+    [second, ..] -> Ok(second)
+    _ -> Error(Nil)
+  }
+  let leading = case values {
+    [_, leading, ..] -> Ok(leading)
+    _ -> Error(Nil)
+  }
```

Comment:
I think this could just be written like this:
```gleam
let assert [second, leading, ..] = values
```

Reply:
Ah yes, good point!

I went that way initially, but was confused with the additional machinery around unwrapping the `leading` value.